code/game/gamemodes/dynamic/dynamic_rulesets_midround.dm

```diff
@@ -171,17 +171,24 @@
 	cost = 10
 	requirements = list(50,40,30,20,10,10,10,10,10,10)
 	repeatable = TRUE
-	flags = TRAITOR_RULESET
 
 /datum/dynamic_ruleset/midround/autotraitor/acceptable(population = 0, threat = 0)
 	var/player_count = mode.current_players[CURRENT_LIVING_PLAYERS].len
 	var/antag_count = mode.current_players[CURRENT_LIVING_ANTAGS].len
 	var/max_traitors = round(player_count / 10) + 1
-	if ((antag_count < max_traitors) && prob(mode.threat_level))//adding traitors if the antag population is getting low
-		return ..()
-	else
+
+	// adding traitors if the antag population is getting low
+	var/too_little_antags = antag_count < max_traitors
+	if (!too_little_antags)
+		message_admins("DYNAMIC: Too many living antags compared to living players ([antag_count] living antags, [player_count] living players, [max_traitors] max traitors)")
```

Comment:
antags -> traitors

Though, do we need these admin messages? Will any admin benefit from constant updates about failures like this? I say we log only. If anything make the log more available via the gamemode panel as an alternative.

Reply:
There's no point to it, no. I'll just move all autotraitor stuff to game.log, including the original "Checking if we can turn someone into a traitor".

Reply:
Are you sure it's antags, though?

`var/antag_count = mode.current_players[CURRENT_LIVING_ANTAGS].len`

Reply:
I guess antags is correct, yeah